databricks/koalas/frame.py

```diff
@@ -228,21 +228,21 @@
 
 >>> df // 10
            angles  degrees
-circle          0       36
-triangle        0       18
-rectangle       0       36
+circle        0.0     36.0
+triangle      0.0     18.0
+rectangle     0.0     36.0
 
 >>> df.floordiv(10)
            angles  degrees
-circle          0       36
-triangle        0       18
-rectangle       0       36
+circle        0.0     36.0
+triangle      0.0     18.0
+rectangle     0.0     36.0
```

Comment:
For `floordiv` and `rfloordiv`, the result will always float since we cannot predict the result type of each column which is already determined before Spark executes the job.

Reply:
In short, since there is a possibility that the result can be a `Infinity` which is `float`, the result always be a `float`.

Someone can give me an opinion about this?